cogs/helpers.py

```diff
@@ -555,6 +555,31 @@ def __init__(self, bot):
         embed = discord.Embed(colour=0xDA291C, description=msg)
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=["color"])
+    async def colour(self, ctx, *, arg: str):
+        """Shows a small image filled with the given hex colour.
+        Usage: `?colour hex`
+        """
+        allowed = re.compile("^#?(?:0x)?([0-9a-fA-F]{6})$")
+        match = allowed.match(arg)
+        if not match:
+            await ctx.send("Please use a valid 6-digit hex number.")
+            return
+        await ctx.trigger_typing()
+        c = int(match.group(1), 16)
+        r = (c & 0xFF0000) >> 16
+        g = (c & 0xFF00) >> 8
+        b = c & 0xFF
+        SIZE = 64
+        img = np.zeros((SIZE, SIZE, 3), np.uint8)
+        img[:, :] = (b, g, r)
+        ext = "jpg"
+        retval, buffer = cv2.imencode('.{}'.format(ext), img,
+                                      [cv2.IMWRITE_JPEG_QUALITY, 100])
```

Comment:
I ran some tests and we get a very minor speedup by sending a lower quality JPEG. I'd crank down the quality to 1.
```suggestion
                                      [cv2.IMWRITE_JPEG_QUALITY, 1])
```

Reply:
bce0d09